### PR TITLE
Adds makeshift grenades

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
gives craftable, dangerous, unreliable grenades to the random fellows of the CEV Eris!
image

## Changelog
:cl:
add: Teaches vagabonds how to blow themselves up easier with emergency tanks and plasma tanks
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
I think you should remove the plasma tank bomb. HE grenades shouldn't be easily available to the public. Also does making the explosive range 1.5 even do anything? It's based on tiles, I think that's just memecode. Either way, I recommend either removing the grenade entirely, or at very least giving it only a light range of 2 and nothing else.

Frag grenade seems ok.

**Comment by AgentMA1** [comment]
That's a VERY cheap HE grenade. Too cheap

File: code/game/objects/items/weapons/grenades/explosive.dm [hunk 1]
```diff
@@ -31,3 +31,15 @@
...
+
+/obj/item/weapon/grenade/explosive/ptank 
+    name = "plasma tank bomb"
+    desc = "A large, dangerous bomb. A weapon for the brave, foolhardy, or desperate."
+    w_class = ITEM_SIZE_NORMAL
+    icon_state = "explosive_ptank"
+    item_state = "explosive_ptank"
+    throw_speed = 2
+    throw_range = 5
+    heavy_range = 0.5
```
Comment (Jackofashepard) [inline code/game/objects/items/weapons/grenades/explosive.dm:43]: 1

**Comment by Jackofashepard** [comment]
Looks fine to me otherwise, fair trade for a grenade that can potentially blow up in your face.

**Comment by Sparrowbird611** [comment]
> Looks fine to me otherwise, fair trade for a grenade that can potentially blow up in your face.

It works like a normal grenade though.

**Comment by CeUvi** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Firefox13** [comment]
..but aren't these literally onetank bombs knock-offs that we have already?

**Comment by Firefox13** [comment]
We already have onetank bombs and these are just a cheap knock-off. Plus, you can even make them with empty tanks as there's no check for gas which is wack. So no.
Good PR though, really

**Pull request closed without merging**